db/migrate/20190226003449_add_poll_id_to_statuses.rb

```diff
@@ -0,0 +1,5 @@
+class AddPollIdToStatuses < ActiveRecord::Migration[5.2]
+  def change
+    add_column :statuses, :poll_id, :bigint
```

Comment:
Same goes here, we can use a foreign key.

Reply:
I think foreign keys are an increasingly bad idea. It requires having an index on that column, otherwise DELETE or NULLIFY cascades take an incredible amount of time. Any index on statuses would also be massive.